### ISSUE TITLE
VHDL beautification

### ISSUE THREAD

**Issue by JYH25** [Remillard/VHDL-Mode#167]
Sorry to bother. I am confused about which package I should download to make VHDL code highlight like yours.

![VHDL highlight](https://user-images.githubusercontent.com/93721016/140331534-99f5c37c-b6ba-4806-9d00-cd0d828b4c81.png)

**Comment by Remillard** [comment]
Well... I made that a LONG time ago.  I have no actual memory of what color package I was using for that.  However I went to my current list of color schemes and just used the up/down arrow and moved through them.  I *think* that this is probably "Monokai"
![image](https://user-images.githubusercontent.com/28012313/140344607-9127d17b-d5fb-4116-9bd5-e57fd277ba31.png)
It has the italicized port names, the red mode, and the italicized type indicator in cyan.  

I don't currently use Monokai though as it's a little too brown for my taste.  Some current favored scheme bundles:
* Agila (has lot of variations, including a Monokai with extended capability and others)
* ayu (only 3 but covers dark, darkblue, and light).

I current use "ayu-mirage":
![image](https://user-images.githubusercontent.com/28012313/140347773-0e0bd2dd-9522-4c0c-8aa0-07a0f9617ceb.png)


Hope that helps!
